sync/class.jetpack-sync-wp-replicastore.php

```diff
@@ -674,6 +640,31 @@ function checksum_histogram( $object_type, $buckets, $start_id = null, $end_id =
 		return $histogram;
 	}
 
+	private function table_checksum( $table, $columns, $id_column, $where_sql = '1=1', $min_id = null, $max_id = null ) {
+		global $wpdb;
+
+		// sanitize to just valid MySQL column names
+		$columns_sql = implode( ',', preg_grep ( '/^[0-9,a-z,A-Z$_]+$/i', $columns ) );
```

Comment:
Great point! I just added code which detects missing columns and returns a `WP_Error`